paddle/cinn/ir/schedule/ir_schedule.cc

```diff
@@ -717,6 +718,448 @@ Expr ScheduleImpl::Rfactor(const Expr& rf_loop, int rf_axis) {
   return rf_create.CreateRfAllStmts();
 }
 
+// Used in FactorizeReduction
+Tensor CreateRFTensor(const Tensor& original_tensor,
+                      const Expr& rf_loop,
+                      int rf_axis) {
+  std::string name = original_tensor->name + "_rf";
+  std::vector<Expr> new_shape = original_tensor->shape;
+  new_shape.insert(new_shape.begin() + rf_axis, rf_loop.As<For>()->extent);
+  Tensor rf_tensor = _Tensor_::Make(name,
+                                    original_tensor->type(),
+                                    new_shape,
+                                    new_shape,
+                                    original_tensor->operation,
+                                    original_tensor->reduce_axis);
+  rf_tensor->WithBuffer("global", name, original_tensor->type());
+  return rf_tensor;
+}
+
+class ReduceBlockCreater {
```

Comment:
Since naming "Reduce Block" is every common in CINN, could you add one sentence comment for `ReduceBlockCreater` that it is used to create reduce block in FactorizeReduction

Same for RBBlockCreater, add a comment that it is used for writing back

Reply:
done